core/src/lib.rs

```diff
@@ -25,4 +25,5 @@ pub mod prelude;
 pub mod ser;
 pub mod types;
 
+pub use enc::*;
```

Comment:
TODO: make sure this follows conventions of the crate

Reply:
This will re-export all public members of the `enc` module. Which currently re-exports all public members of address and base.

Can we move this into `prelude` tho? I've been meaning to drop the re-exports from lib in favor of a single place

Reply:
Moved into `prelude`